code/tarakanov_kirill_lab1/model/modify_stack.py

```diff
@@ -0,0 +1,76 @@
+class ModifyStack:
+    """
+    Class ModifyStack implements a data structure modified stack
+    Stack works works with int and float only
+    The following operations are supported:
+    * Insertion O(1)
+    * Deletion top element O(1)
+    * Read top element O(1)
+    * Find min element in stack O(1)
+    :attribute _stack: stores elements in stack
+    :attribute _min_stack: stores min_elements for each stack state
+    """
+
+    def __init__(self, init_list=None):
+        if init_list is None:
+            self._stack = []
+            self._min_stack = []
+        else:
```

Comment:
Maybe you should put it on 1st position and not dublicate self._stack = [] self._min_stack = []